src/main/webapp/form.html

```diff
@@ -25,7 +25,10 @@ <h3>a recommendation system for students, by students!</h3>
       <p>and here is my profile picture!</p>
       <input type="file" name="user-img" accept="image/*">
       <br>
-      <input class="button" type="submit" />
+      <!-- <input class="button" type="submit" /> -->
```

Comment:
remove please

Reply:
This is the real implementation since we will need this to call the server, should we remove this now? This is commented out for now and is replaced by a simple button to make it easier to navigate through the pages.

Reply:
Thanks for the explanation! yes please let's remove it now and add it later when we need it :)